examples/kubernetes-mixin-example/porter.yaml

```diff
@@ -13,6 +13,11 @@ invocationImage: deislabs/porter-kubernetes:latest
 tag: deislabs/porter-kube-bundle:v0.1.0
 
 install:
+  - exec:
```

Comment:
Did you intend to add this permanently to the example? Or was this accidentally included from our debugging session last night?

Reply:
embarassed emoji.....

Reply:
let me go clean that up :)